src/components/Move/Move.module.scss

```diff
@@ -0,0 +1,149 @@
+.Move {
+  width: 280px;
+  border: 0.5px solid #373e46;
+  border-radius: 10px;
+  background: #282e33;
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  padding: 10px;
+  position: absolute;
+  left: 0;
+  top: -500px;
+  z-index: 120;
+
+  .title {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

Comment:
display: flex;
    align-items: center;
    justify-content: center;
    
    make mixin for this